src/identity/ed25519/ed25519-key-pair-identity.ts

```diff
@@ -30,7 +30,15 @@ export class Ed25519KeyPairIdentity extends PublicKeyIdentity {
       throw new Error("Invalid Mnemonic")
     }
     const seed = bip39.mnemonicToSeedSync(sanitized).slice(0, 32)
-    const keys = ed25519.generateKeyPair({ seed })
+    return Ed25519KeyPairIdentity.fromSeed(seed)
+  }
+
+  static fromSeed(seed: string | pki.ed25519.NativeBuffer) {
```

Comment:
Hmm... `seed` in this case is just the private key in hexadecimal format. Should we call it `fromHex` instead?

Reply:
renamed to `fromHex`